sdk/translation/Azure.AI.Translation.Text/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "AZC0012:Avoid single word type names", Justification = "Class names are generated based on the Translator API.")]
```

Comment:
This should be removed

Reply:
We do not plan to change this as was agreed during the APIView.